app/helpers/remote_execution_helper.rb

```diff
@@ -74,6 +74,15 @@ def template_invocation_status(task)
     end
   end
 
+  def template_invocation_actions(task, host)
+    if task.nil?
+      []
+    else
+      [display_link_if_authorized(_("Details"), hash_for_template_invocation_path(:id => task).merge(:auth_object => host, :permission => :view_foreman_tasks))]
+    end
+
```

Comment:
It seems even the Ruby Police is corrupt: how coule Rubocop let you go trying to smuggle this empty line?